FlowCrypt/Controllers/Attachment/AttachmentViewController.swift

```diff
@@ -139,7 +139,7 @@ final class AttachmentViewController: UIViewController {
     private func showTextAttachment() {
         view.addSubview(textView)
         view.constrainToEdges(textView)
-        textView.text = file.data.toStr()
+        textView.text = file.data?.toStr()
```

Comment:
I made `data` property optional for `MessageAttachment` as it'll be `nil` when only metadata of attachment is fetched.
On this screen `data` should always be not `nil`, as screen is opened only after full attachment is fetched.

Reply:
Then you could add `?? "(error: failed to load this message. please report this issue)"` so that if it ever breaks, it becomes more obvious and when users report it, we'll know right away